libafl/src/stages/mutational.rs

```diff
@@ -108,12 +105,6 @@ where
     ) -> Result<(), Error> {
         start_timer!(state);
 
-        // Here saturating_sub is needed as self.iterations() might be actually smaller than the previous value before reset.
```

Comment:
I still think we should leave this in as option: if I create this stage with 1000 iterations, I don't want it to sometimes stop after 50 iterations. It's simply less correct ATM.

Reply:
btw this guy is already deleted in #2331

Reply:
and in reality if you do in-process fuzzing you do want to stop fuzz this testcase immediately and just discard this testcase.

restarting fuzzers over and over again is a huge loss of time

Reply:
i don't know calling saturating_sub would make this more correct.

Imagine, for the first cycle, this self.iteration() returned 1000. so you fuzz it for 1000 times.
and it crashed at first. then there're 999 times remaining.

The next time you restart, self.iteration() returned 100. you saturate_sub() then it becomes 0.
I don' think this is more correct